### PR TITLE
Add note about wrapped staking messages

### PR DESCRIPTION
**Pull request by gusin13**
Validators are getting confused and using the normal staking messages to delegate/unbond etc due to which their txs are stuck in mempool. Add proper notes to warn them about our special case and the delayed wrapped messages.

Screenshot 2024-02-04 at 8 06 03 PM
Screenshot 2024-02-04 at 8 06 50 PM
Screenshot 2024-02-04 at 8 06 54 PM

### CODE REVIEW

File: docs/cli/babylond/Babylond_tx/Babylond_tx_staking/Babylond_tx_staking_create-validator.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-sidebar_label: babylond tx stking create-validator
+sidebar_label: babylond tx staking create-validator
 hide_table_of_contents: true
 
 ---
 
+:::warning
+Use the wrapped message 
+[create-validator](/docs/cli/babylond/Babylond_tx/Babylond_tx_checkpointing/babylond_tx_checkpointing_create-validator) in the 
+checkpointing module.
```
Comment (SebastianElvis) [inline docs/cli/babylond/Babylond_tx/Babylond_tx_staking/Babylond_tx_staking_create-validator.md:11]:
```suggestion
This message is disabled in Babylon. Please use the wrapped message 
[create-validator](/docs/cli/babylond/Babylond_tx/Babylond_tx_checkpointing/babylond_tx_checkpointing_create-validator) in the 
checkpointing module instead.
```

Same comments for other appearances.

File: docs/user-guides/btc-staking-testnet/become-validator.md [hunk 2]
```diff
@@ -123,3 +123,9 @@ on the `create-validator` transaction.
 After the epoch ends and if you have enough stake to be an active validator,
 performing this query will return you a status `BOND_STATUS_BONDED`.
 Congrats! You are now a validator on the Babylon system.
+
+:::info
+If you want to delegate, redelegate, unbond or cancel-unbond use the wrapped messages in the
```
Comment (SebastianElvis) [inline docs/user-guides/btc-staking-testnet/become-validator.md:128]:
```suggestion
If you want to delegate, redelegate, unbond or cancel-unbond, please use the wrapped messages in the
```

**Review by SebastianElvis — APPROVED** [review]
Lgtm!

File: docs/user-guides/btc-timestamping-testnet/become-validator.md [hunk 3]
```diff
@@ -149,3 +149,9 @@ on the `create-validator` transaction.
 After the epoch ends and if you have enough stake to be an active validator,
 performing this query will return you a status `BOND_STATUS_BONDED`.
 Congrats! You are now a validator on the Babylon system.
+
+:::info
+If you want to delegate, redelegate, unbond or cancel-unbond use the wrapped messages in the
```
Comment (gitferry) [inline docs/user-guides/btc-timestamping-testnet/become-validator.md:154]:
```suggestion
If you want to delegate, redelegate, unbond or cancel-unbond, please use the wrapped messages in the
```

**Review by gitferry — APPROVED** [review]

File: docs/cli/babylond/Babylond_tx/Babylond_tx_staking.md [hunk 4]
```diff
@@ -5,22 +5,40 @@ hide_table_of_contents: true
 
 ---
 
+:::info
+Please use the wrapped messages in the [epoching](/docs/cli/babylond/Babylond_tx/Babylond_tx_epoching) and 
+[checkpointing](/docs/cli/babylond/Babylond_tx/Babylond_tx_checkpointing) module to perform
+staking-related operations.
+
+In order to keep the validator set unchanged in the middle of epochs, the epoching
+module intercepts and rejects staking-related messages that affect validator sets via
```
[the remark this answers is not in the document]
Comment (gusin13) [reply docs/cli/babylond/Babylond_tx/Babylond_tx_staking.md:14]: It is mentioned in our epoching module Readme that Antehandler rejects the staking messages?
https://github.com/babylonchain/babylon/tree/dev/x/epoching#babylons-epoching-module-design

**Pull request merged**